Description: Don't output pip warnings, otherwise some doctests fail
 "--format=legacy" suppresses a warning that the default format will
 change from legacy to columns. This is being discussed in
 https://trac.sagemath.org/ticket/21835
Author: Tobias Hansen <thansen@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/src/sage/misc/package.py
+++ b/sage/src/sage/misc/package.py
@@ -144,7 +144,7 @@
         sage: d['beautifulsoup']   # optional - beautifulsoup
         '...'
     """
-    proc = subprocess.Popen(["pip", "list", "--no-index"], stdout=subprocess.PIPE)
+    proc = subprocess.Popen(["pip", "list", "--no-index", "--format=legacy"], stdout=subprocess.PIPE)
     stdout = str(proc.communicate()[0])
     return dict((name.lower(), version) for name,version in PIP_VERSION.findall(stdout))
 
